src/main/resources/assets/origins/lang/zh_cn.json

```diff
@@ -1,125 +1,199 @@
 {
     "origin.origins.human.name":"人类",
     "origin.origins.human.description":"普通的人类。普通的Minecraft体验等着你。",
+    
     "origin.origins.merling.name":"人鱼族",
-    "origin.origins.merling.description":"海洋的原住民。他们并不习惯长时间地离开水体。",
```

Comment:
`"origin.origins.merling.description"`: `"These natural inhabitants of the ocean are not used to being out of the water for too long."`